extensions/amp-story/1.0/media-pool.js

```diff
@@ -552,7 +552,9 @@ export class MediaPool {
           poolMediaEl,
           new UpdateSourcesTask(this.win_, sources)
         );
-        this.enqueueMediaElementTask_(poolMediaEl, new LoadTask());
+        this.enqueueMediaElementTask_(poolMediaEl, new LoadTask()).then(() => {
+          ampMediaForDomEl.dispatchCustomEvent(VideoEvents.LOAD);
```

Comment:
We should call the `onVideoLoaded` in the same fashion that we call `resetOnDomChange`:
```
ampMediaForDomEl.getImpl().then((impl) => {
      if (impl.onVideoLoaded) {
        impl.onVideoLoaded();
      }
    });
```